src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+finger) that is large enough to trigger a scroll and the first paint which reflects the scrolling
+taking place.
+
+### Aggregate all interactions per page
+
+Once we’ve defined what the latency of an interaction is, we’ll need to compute an aggregate value
+for a page load, which may have many user interactions. Having an aggregated value enables us to:
+
+* Form correlations with business metrics.
+* Evaluate correlations with other performance metrics. Ideally, our new metric will be sufficiently
+  independent that it adds value to the existing metrics.
+* Easily expose values in tooling in ways that are easy to digest.
+
+In order to perform this aggregation we need to solve two questions:
+
+1. What numbers do we try to aggregate?
+2. How do we aggregate those numbers?
+
+We’re exploring and evaluating several options. We welcome feedback on this aggregation.
+
+One option is to define a budget for the latency of an interaction, which may depend on the type
+(scroll, keyboard, tap, or drag). So for example if the budget for taps is 100&nbsp;ms and the
+latency of a tap is 150&nbsp;ms then the amount over budget for that interaction would be
+50&nbsp;ms. Then we could compute the maximum amount of latency that goes over the budget for any
+user interaction in the page.
+
+Another option is to compute the average or median latency of the interactions throughout the life
+of the page. So if we had latencies of 80&nbsp;ms, 90&nbsp;ms, and 100&nbsp;ms, then the average
+latency for the page would be 90&nbsp;ms. We could also consider the average or median ‘over budget’
+to account for different expectations depending on the type of interaction.
+
+## How does this look like on web performance APIs?
+
+### What's missing from Event Timing?
+
+Unfortunately not all of the ideas presented in this post can be captured using the Event Timing
+API. In particular, there’s no simple way to know the events associated with a given user
+interaction with the API. In order to do this, we’ve [proposed adding an `interactionID` to the
+API](https://docs.google.com/presentation/d/1nxNFwsGqYy7WmIZ3uv_0HsSIQMSXQA9_PqlOD3V74Us/edit#slide=id.p).
+
+Another shortcoming of the Event Timing API is that there is no way to measure the scroll
+interaction, so we’re [working on enabling these
+measurements](https://docs.google.com/presentation/d/1qVdMlqgi9uuyx9imCauzMjLGHQK6TGOIZV_RnlGBKis/edit)
+(via Event Timing or a separate API).
+
+### What can you try right now?
+
+Right now, it is still possible to compute the maximum latency for taps/drags and for keyboard
+interactions. The following code snippet would produce these two metrics.
+
+```js
+let maxTapOrDragDuration = 0;
+let maxKeyboardDuration = 0;
+const observer = new PerformanceObserver(list => {
+  list.getEntries().forEach(entry => {
+    switch(entry.name) {
+      case "keydown":
+      case "keyup":
+        maxKeyboardDuration = Math.max(maxKeyboardDuration,
```

Comment:
This is returns a _normalized max_, across all interactions?  I worry that this syntax may confuse folks to assume its the max across one interaction...

I know you highlight that interactionID is not available, but, I wonder if therefore it's better to just console.log all the durations instead of printing a single max?

(Maybe that also depends if you take my suggestion to drop aggregation topic from the post)

Reply:
This is the actionable bit from the page-aggregation section :) It's easy to log all durations, but less clear to me how you'd convert those into a number.